packages/rum/test/e2e/react/index.html

```diff
@@ -2,11 +2,12 @@
 <html>
 
 <head>
-  <title>test</title>
+  <title>React test</title>
   <base href="/test/e2e/react/">
-  <meta http-equiv="Content-Security-Policy" content="default-src 'self'; connect-src http://localhost:8000 http://localhost:8200 https://frontend-test-apm-server.elastic.co">
+  <meta http-equiv="Content-Security-Policy" content="default-src 'self';img-src 'self' data:; connect-src http://localhost:8000 http://localhost:8200 http://localhost:8001">
```

Comment:
lets remove this line. Feels leeky. If we want to do this right, then we have to inject it from server with the variables configured for server URL. This is the reason why I was standardising all the test configs in my PR #187

Reply:
We need to make sure that the agent works when A CSP is present in the page! so this is not for  adding A CSP to secure our tests, it is for testing if the agent works properly if the CSP is in the page! Let's keep these until we have added it from the server. Should be a separate PR IMO.

Reply:
Sure. makes sense